crawl-ref/source/describe.cc

```diff
@@ -1095,12 +1095,6 @@ static void _append_weapon_stats(string &description, const item_def &item)
     const bool could_set_target = _could_set_training_target(item, true);
 
     if (skill == SK_SLINGS)
-    {
-        description += make_stringf("\nFiring bullets:    Base damage: %d",
```

Comment:
You need to remove the entire `if` statement here; in its current state this code does not work as intended.